datahub-web-react/src/app/entity/shared/containers/profile/sidebar/Ownership/AddOwnerModal.tsx

```diff
@@ -228,7 +232,7 @@ export const AddOwnerModal = ({ urn, type, visible, hideOwnerType, defaultOwnerT
                         tagRender={(tagProps) => <Tag>{tagProps.value}</Tag>}
                     >
                         {combinedSearchResults?.map((result) => (
-                            <Select.Option value={result.entity.urn}>{renderSearchResult(result)}</Select.Option>
+                            <Select.Option value={result.entity.urn}> {renderSearchResult(result)}</Select.Option>
```

Comment:
Do we need a key here also?

Reply:
Yes, we do required the key property here. We have added the separate function in which will help to generate the unique key for the datasets of combinedSearchResults.